htdocs/class/module.textsanitizer.php

```diff
@@ -316,8 +316,11 @@ public function makeClickable($text)
     {
         $pattern = "/(^|[^]_a-z0-9-=\"'\/:\.])([-_a-z0-9\'+*$^&%=~!?{}]++(?:\.[-_a-z0-9\'+*$^&%=~!?{}]+)*+)@((?:(?![-.])[-a-z0-9.]+(?<![-.])\.[a-z]{2,6}|\d{1,3}(?:\.\d{1,3}){3})(?::\d++)?)/i";
         $text = preg_replace_callback($pattern, 'self::makeClickableCallbackEmailAddress', $text);
```

Comment:
BTW - the line 318 should be deleted:
```php
$text = preg_replace_callback($pattern, 'self::makeClickableCallbackEmailAddress', $text);
```
since this is covered below it